### ISSUE TITLE
events not fire

### ISSUE THREAD

**Issue by windydang26** [TradingPal/react-native-highcharts#62]
i cannot catch event. someone any suggest?

**Comment by salujaharkirat** [comment]
events to native code?

**Comment by windydang26** [comment]
yes! i'm using react-native. events still running, but seem is web enviroment. i cannot consolog, only alert() as web. how to transmission params of hightchart to component?

**Comment by salujaharkirat** [comment]
Webview component injects a function called `postMessage` in your webview html when you pass `onMessage` prop to it.  You can use this `postMessage` to fire events from webview.

You can try something like this [https://snack.expo.io/Bycv3CnMz](https://snack.expo.io/Bycv3CnMz)

**Comment by windydang26** [comment]
oh! thank you very much! i will try it :)

**Comment by windydang26** [comment]
@jinxac Webview component in library not contain onMessage props. i fork repo and edit but cannot  fire onMessage. i want to fire event to receiver data point when click in highchart? are you have other idea? thank.

**Comment by salujaharkirat** [comment]
are u trying this on android @windydang26 ?

**Comment by windydang26** [comment]
so sorry. i forget method window.postMessage. after i use postMessage to send data and onMessage to receiver data as you told. i fired events. hi. thank you @jinxac :)

**Comment by cstpraveen** [comment]
Hi try to change from `source={{ html: newHTML, baseUrl: 'web/' }}` to `source={{ html: newHTML, baseUrl: 'http://localhost' }}`.

It will work, or else try to remove baseUrl from the source attribute.
